tezos/protocols.go

```diff
@@ -126,10 +126,10 @@ var (
 			{Proxford, 0, 16385, -1, 2, 3, 8192, 512},  // v18
 		},
 		Parisnet: {
-			{ProtoGenesis, 0, 0, 0, 0, 3, 8192, 512},     // 0
-			{ProtoBootstrap, 0, 1, 1, 0, 3, 8192, 512},   // 0
-			{Proxford, 2, 2, 16384, 0, 3, 8192, 512},     // v18
-			{PtParisB, 0, 16385, -1, 2, 2, 12288, 12288}, // v19
+			{ProtoGenesis, 0, 0, 0, 0, 3, 8192, 512},    // 0
+			{ProtoBootstrap, 0, 1, 1, 0, 3, 8192, 512},  // 0
+			{Proxford, 2, 2, 8192, 0, 3, 8192, 512},     // v18
+			{PtParisB, 0, 8193, -1, 2, 2, 12288, 12288}, // v19
```

Comment:
```suggestion
			{PtParisB, 0, 8193, -1, 1, 2, 12288, 12288}, // v19
```
[Ref](https://rpc.paris2net.teztnets.com/chains/main/blocks/8193)